src/.eslintrc.yml

```diff
@@ -7,6 +7,7 @@ env:
 
 parserOptions:
   ecmaVersion: 2016
+  sourceType: module
```

Comment:
eslint warns me `import` is illegal in non-modules

Reply:
oh right, `index.js` is a module now! 🤪